modules/caddyhttp/replacer_test.go

```diff
@@ -171,6 +171,10 @@ eqp31wM9il1n+guTNyxJd+FzVAH+hCZE5K+tCgVDdVFUlDEHHbS/wqb2PSIoouLV
 			input:  "{http.request.tls.client.san.ips.0}",
 			expect: "127.0.0.1",
 		},
+		{
+			input:  "{http.request.tls.client.certificate_pem}",
+			expect: "<empty>",
```

Comment:
This doesn't seem like the right expected result... it should be the same as the `clientCert` var defined above.

Reply:
Fixed. Although the PEM encoded value comes with a newline attached due to which tests were failing.